### PR TITLE
Ana/small fix in totalrewards

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Just a very small fix. ModalWithdrawRewards was throwing an `amount of undefined` error when no more rewards where left.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/ActionModal/components/ModalWithdrawRewards.vue [hunk 1]
```diff
@@ -107,7 +111,9 @@ export default {
         )
         return stakingDenomBalance
           ? stakingDenomBalance.denom
-          : nonZeroBalances.length > 0 ? nonZeroBalances[0].denom : this.denom
+          : nonZeroBalances.length > 0
+          ? nonZeroBalances[0].denom
```
Comment (faboweb) [inline src/ActionModal/components/ModalWithdrawRewards.vue]: indent

File: src/ActionModal/components/ModalWithdrawRewards.vue [hunk 2]
```diff
@@ -71,7 +71,11 @@ export default {
         .filter(({ denom }) => denom === this.denom)
         .reduce((sum, { amount }) => sum + Number(amount), 0)
         .toFixed(6)
-      return stakingRewards > 0 ? stakingRewards : this.claimedReward.amount
+      return stakingRewards > 0
+        ? stakingRewards
+        : this.claimedReward
+        ? this.claimedReward.amount
```
Comment (faboweb) [inline src/ActionModal/components/ModalWithdrawRewards.vue]: indent

Comment (faboweb) [reply src/ActionModal/components/ModalWithdrawRewards.vue]: Does it revert to this formatting or does it keep the indentation on lint fix? I don't think this is good to read.

Comment (Bitcoinera) [reply src/ActionModal/components/ModalWithdrawRewards.vue]: I think this is the same indentation we had before no? I will just refactor

Comment (faboweb) [reply src/ActionModal/components/ModalWithdrawRewards.vue]: don't worry. I think eslint is forcing this

**Pull request merged**